commit 499900990c8e9432222df04b462bf8a686a17f49 (refs/changes/24/2824/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-10-19T15:24:17-07:00 (2 years ago)
    
    TOOLS-1901 sdcadm build fails if there is no `node` on the PATH

diff --git a/Makefile b/Makefile
index 05d06d2..2dc9162 100644
--- a/Makefile
+++ b/Makefile
@@ -53,8 +53,8 @@ endif
 .PHONY: all
 all: | $(NPM_EXEC)
 	MAKE_OVERRIDES='CTFCONVERT=/bin/true CTFMERGE=/bin/true' $(NPM) install
-	./node_modules/.bin/kthxbai || true # work around trentm/node-kthxbai#1
-	./node_modules/.bin/kthxbai
+	$(NODE) ./node_modules/.bin/kthxbai || true # work around trentm/node-kthxbai#1
+	$(NODE) ./node_modules/.bin/kthxbai
 	rm -rf ./node_modules/.bin/kthxbai ./node_modules/kthxbai
 
 .PHONY: shar
